content/en/docs/reference/config/telemetry/configurable_metrics/index.md

```diff
@@ -1,11 +1,7 @@
 ---
```

Comment:
Is this doc going to have more content in the future? If not, why not delete it and add an alias for it in the new task?

Reply:
I think so. But we might want "configurable telemetry" as a title or something like that, since this should cover tracing (cc @jacob-delgado @nrjpoddar ).

Reply:
Agree with @frankbu, we can either delete this page or rename it to "Configure Telemetry" like @kyessenov said and add link to tracing docs which is https://preliminary.istio.io/docs/tasks/observability/distributed-tracing/configurability/

Reply:
Done